.release/ci.hcl

```diff
@@ -0,0 +1,270 @@
+schema = "1"
+
+project "consul-k8s" {
+  team = "consul-k8s"
+  slack {
+    notification_channel = "CBXF3CGAF" # team-consul-kubernetes
+  }
+  github {
+    organization = "hashicorp"
+    repository = "consul-k8s"
+    release_branches = [
+      # The CRT tool does not support * yes as a branch name
+      "main",
+      "cb/crt-testing"
+    ]
+  }
+}
+
+event "merge" {
+  // "entrypoint" to use if build is not run automatically
+  // i.e. send "merge" complete signal to orchestrator to trigger build
+}
+
+event "build" {
+  depends = ["merge"]
+  action "build" {
+    organization = "hashicorp"
+    repository = "consul-k8s"
+    workflow = "build"
+  }
+}
+
+event "upload-dev" {
+  depends = ["build"]
+  action "upload-dev" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "upload-dev"
+    depends = ["build"]
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "promote-dev-docker" {
```

Comment:
for readability, can we move this event definition to be after the `verify` definition